public/manifest.cache

```diff
@@ -0,0 +1,4 @@
+CACHE MANIFEST
```

Comment:
I haven't ever used a cache manifest before. Reading up on it, [this document](https://web.dev/articles/appcache-beginner) makes me wonder if we need to reference this somewhere (like in the `<html manifest="">`). Is that right?